docs/concepts/gas.md

```diff
@@ -224,8 +224,6 @@ _So how can a developer pay the gas fee for their users on NEAR?_
 
 A user can use the funds directly from the developers account suitable only for the gas fees on this dApp. Then the developer has to distinguish users based on the signers' keys instead of the account names.
```

Comment:
```suggestion
- A user can use the funds directly from the developer's account suitable only for the gas fees on this dApp. Then the developer has to distinguish users based on the signers' keys instead of the account names.
```